app/Libraries/OsuMarkdownProcessor.php

```diff
@@ -92,11 +93,17 @@ public static function process($rawInput, $config)
 
     public static function parseYamlHeader($input)
     {
-        $hasMatch = preg_match('#^(?:---\n(?<header>.+?)\n(?:---|\.\.\.)\n)(?<document>.+)$#s', $input, $matches);
+        $hasMatch = preg_match('#^(?:---\n(?<header>.+?)\n(?:---|\.\.\.)\n)(?<document>.+)$#', $input, $matches);
```

Comment:
This seems to fail to parse the news headers since there's usually more than one new line after the ending `---`

Reply:
got confused between /g and /s :dancer: